cmd/traefik/traefik.go

```diff
@@ -239,6 +239,9 @@ func setupServer(staticConfiguration *static.Configuration) (*server.Server, err
 
 	pluginBuilder, err := createPluginBuilder(staticConfiguration)
 	if err != nil {
+		if staticConfiguration.Experimental != nil && staticConfiguration.Experimental.RequirePluginsToStart {
```

Comment:
I would suggest another name for the option, I somehow would find the following more clear:

```suggestion
		if staticConfiguration.Experimental != nil && staticConfiguration.Experimental.AbortOnPluginFailure {
```